plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/active-filters/index.tsx

```diff
@@ -10,11 +10,12 @@ import { productFilterActive } from '@woocommerce/icons';
  */
 import metadata from './block.json';
 import Edit from './edit';
-import './style.scss';
+import Save from './save';
 
 if ( isExperimentalBlocksEnabled() ) {
 	registerBlockType( metadata, {
 		icon: productFilterActive,
 		edit: Edit,
+		save: Save,
```

Comment:
Save is not a component but a function. it shouldn't be in Camelcase.

```suggestion
		save,
```